src/parser/content-evaluator-module.ts

```diff
@@ -133,14 +126,60 @@ export class ContentEvaluatorModule implements Module {
     }, {});
   }
 
+  _splitCommentsByPrompt(commentsWithScore: Readonly<GithubCommentScore>[]): {
+    commentsToEvaluate: CommentToEvaluate[];
+    PrCommentsToEvaluate: PrCommentToEvaluate[];
```

Comment:
```suggestion
    prCommentsToEvaluate: PrCommentToEvaluate[];
```

Reply:
Shouldn't this be spotted by `eslint`?

Reply:
I think the rule is pascal case so technically it's pascal case.

Reply:
then the rule should be changed to camelCase

Reply:
If that was the case then the linter would detect `commentsToEvaluate` as wrong. Either it accepts both, either there is no rule for this.

Reply:
It should be camelCase as suggested. I have renamed it.

Currently, in eslint naming convention rule, no rule applies to this. The selector needed for this is `typeProperty` which is absent in the rules,  and allowing camelCase isn't enough either because there are properties in the code with names like "pull_number" and "NFT_ADDRESS".  So, I haven't changed the rules.